frontend/src/v5/ui/controls/sideBar/sidebar.component.tsx

```diff
@@ -23,17 +23,15 @@ interface ISidebar {
 	onClick: () => void;
 	className?: string;
 	children: JSX.Element;
-	hidden?: boolean;
 }
 
 export const Sidebar = ({
 	className,
 	open,
 	onClick,
-	hidden = false,
 	children,
 }: ISidebar): JSX.Element => (
-	<SidebarContainer className={className} open={open} hidden={hidden}>
+	<SidebarContainer className={className} open={open}>
 		<ExpandButton onClick={onClick}>
```

Comment:
Perhaps this is a problem for a new issue. Either way it's only a small bug.
When the sidebar is collapsed you can keep pressing tab until you tab to the ExpandButton. This shifts the contents of the modal to the left and shows a scrollbar
![image](https://user-images.githubusercontent.com/62481250/226429406-99cd25c2-26ec-4c58-931c-aa68e793e76a.png)

This also occurs on staging